sql/src/value.rs

```diff
@@ -30,10 +30,11 @@ use crate::{
     schema::{DecimalDataType, DecimalSize},
 };
 
+use arrow_array::IntervalMonthDayNanoArray;
```

Comment:
this should locate in `#[cfg(feature = "flight-sql")]` below

Reply:
Done

Reply:
We could just move this line down

Reply:
> ould just move this line down

Ok